doc/spec.adoc

```diff
@@ -55,6 +59,28 @@ This field can be used for layered products to enforce their dependencies so tha
 Each element is composed of:
 ** Mandatory `groupId` and `artifactId` elements that are the Maven coordinates of the required channel.
 ** Optional `version` to stick to a given channel version (instead of requiring the latest version of that channel). In the absence of this `version`, the latest version of the channel will be determined based on the Maven repository metadata.
+* A collection of `repositories` that defines repositories associated with the channel. Only listed repositories are used to resolve channel's components. Each repository is composed of:
+** A required `id` that defines the repository name used in Maven cache
+** A required `url` pointing to a default Maven repository
```

Comment:
What do you mean by "default" Maven repository?

I understand it as the "initial" Maven repository that are delivering the artifacts from the channel (eg nexus.repository.jboss.org).

Reply:
I was trying to say this is the URL that will be used if the user doesn't provide their own alternative - like a proxy or alternative repository location